src/test/ui/self/suggest-self-2.rs

```diff
@@ -4,21 +4,18 @@ impl Foo {
     fn foo(&self) {
         bar(self);
         //~^ ERROR cannot find function `bar` in this scope
-        //~| HELP try calling `bar` as a method
```

Comment:
Could you avoid unnecessary changes (this test, formatting in `compiler/rustc_resolve/src/late/diagnostics.rs`)?